ci-operator/step-registry/openshift/e2e/ovirt/clusterbot/openshift-e2e-ovirt-clusterbot-workflow.yaml

```diff
@@ -0,0 +1,24 @@
+workflow:
+  as: openshift-e2e-ovirt-clusterbot
+  steps:
+    pre:
+    - chain: ipi-ovirt-pre
+    - ref: ipi-conf-ovirt-generate-csi-test-manifest
+    test:
+    - as: e2e-test
+      commands: sleep $(CLUSTER_DURATION) & wait
+      from_image:
+        namespace: origin
+        name: centos
+        tag: '8'
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    post:
+    - chain: ipi-ovirt-post
+    leases:
+      - env: OVIRT_UPGRADE_LEASED_RESOURCE
```

Comment:
The env needs to be:
OVIRT_CLUSTERBOT_LEASED_RESOURCE

The `OVIRT_UPGRADE_LEASED_RESOURCE` has specific logic here https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/ovirt/generate-install-config-params/ipi-conf-ovirt-generate-install-config-params-commands.sh#L13

Reply:
I don't quite follow how leases work - does it have to be a unique env var?

Reply:
It basically means that the lease name will be stored in an env var specified by leases.env.
So here what you did means that the lease name will be stored in en env var `OVIRT_UPGRADE_LEASED_RESOURCE` which will be available for all the steps.
On ovirt we have logic that the leases names correlate to secrets we have, that way we isolate openshift clusters from one another on the oVirt engine.
If you will use the `OVIRT_UPGRADE_LEASED_RESOURCE` env var, then the test will look for a secret with that name, which will not be there